src/features/common/components/CallToActionButton/CallToActionButton.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import styles from './CallToActionButton.module.scss';
+
+const CallToActionButton = ({ children, type, path }) => (
+  <Link className={`${styles[type]} ${styles.button}`} type="button" to={path}>
```

Comment:
Do we need the `type` prop?

Reply:
Yes, for assign different colors. If you have better idea, please let me know.